apps/judicial-system/backend/src/app/modules/defendant/civilClaimant.controller.ts

```diff
@@ -79,19 +81,20 @@ export class CivilClaimantController {
   async update(
     @Param('caseId') caseId: string,
     @Param('civilClaimantId') civilClaimantId: string,
+    @CurrentCivilClaimant() civilClaimant: CivilClaimant,
```

Comment:
Can't we remove the line above this one now, since we're accepting the whole civil claimant? We'd also need to change apps/judicial-system/backend/src/app/modules/defendant/test/civilClaimantController/update.spec.ts accordingly

Reply:
This is according to the coding pattern we use in the backend. The @Param decorators retrieve values from the URL and the @Current??? directives retrieve values set by the accompanying guards. Neither directive results in any real work, just assigments to variables, but they act as extra safety checks validatong that the request and the endpoint setup are properly set up.